docs/extensions/EnumValuesExtension.md

```diff
@@ -0,0 +1,30 @@
+---
+title: EnumValuesExtensions
+author: Sergio0694
+description: A markup extension that returns a collection of values of a specific enum type.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, markup extension, XAML, markup, enum
+---
+
+# EnumValuesExtensions
+
+The [`EnumValuesExtensions`](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.EnumValuesExtensions) type implements a markup extension that returns a collection of values of a specific enum type. It can be useful to easily bind a collection of all possible values from a given enum type to a UI element such as a [`ComboBox`](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/combo-box) or some other items container or selector control.
```

Comment:
```suggestion
The [`EnumValuesExtensions`](/dotnet/api/microsoft.toolkit.uwp.ui.EnumValuesExtensions) type implements a markup extension that returns a collection of values of a specific enum type. It can be useful to easily bind a collection of all possible values from a given enum type to a UI element such as a [`ComboBox`](/windows/uwp/design/controls-and-patterns/combo-box) or some other items container or selector control.
```
chop links